src/Console/Command/CheckCommand.php

```diff
@@ -40,6 +41,7 @@ protected function configure()
                     ),
                     new InputOption('no-cache', null, InputOption::VALUE_NONE, 'Disable rule set cache'),
                     new InputOption('cache-dir', null, InputOption::VALUE_REQUIRED, 'Cache directory', '.rules/'),
+                    new InputOption('log-html', null, InputOption::VALUE_OPTIONAL, 'Log output in HTML format to file.'),
```

Comment:
:-1:  If the option is missing, there shouldn't be any HTML output at all. And if I'm not mistaken, this is how the feature is currently implemented.